build.gradle

```diff
@@ -370,6 +372,10 @@ task testConnected(type: Test) {
     testLogging {
         events = ["passed", "failed", "skipped", "started", "standard_out"]
     }
+    retry {
```

Comment:
I wonder if there's a way to share the config between connected and integration.  Not for this PR but maybe worth thinking about at some point

Reply:
Good point!